pybamm/simulation.py

```diff
@@ -358,6 +364,23 @@ def get_experiment_parameter_values(self, op):
             experiment_parameter_values.update(
                 {"Power function [W]": op["Power input [W]"]}
             )
+
+        if op["temperature"] is not None:
+            ambient_temperature = op["temperature"] + 273.15
+            experiment_parameter_values.update(
+                {"Ambient temperature [K]": ambient_temperature}
+            )
+            if op_number == 0:
+                experiment_parameter_values.update(
+                    {
+                        "Initial temperature [K]": ambient_temperature,
+                    }
+                )
+        else:
+            experiment_parameter_values.update(
+                {"Ambient temperature [K]": self._original_temperature}
+            )
```

Comment:
add comment explaining why this needs to be done like this

Reply:
I've added: 

```
            # If at the first operation, then the initial temperature
            # should be the ambient temperature.
```

I thought this was the most intuitive behavior but I guess we don't absolutely have to have the cell start at the ambient temperature. Can either always start at 25oC or take whatever is in the parameter_values? What's your thoughts?